examples/Transports/FIX/over_two_sessions.py

```diff
@@ -39,7 +39,7 @@ def fixmsg(source):
     to serialise and parse messages and pass the codec explicitly.
     """
     # python 2 and 3 compatibility
-    source = {tag: val.encode('utf-8') for tag, val in source.items()}
+    #source = {tag: val.encode('utf-8') for tag, val in source.items()}
```

Comment:
Same comment as above - source is now undefined?

Reply:
removed this line